lib/cdo/http_cache.rb

```diff
@@ -101,6 +101,7 @@ class HttpCache
     hello-world-emoji-2021
     hello-world-space-2022
     hello-world-soccer-2022
+    music-jam-2024
```

Comment:
@breville @sureshc note that [last year](https://github.com/code-dot-org/code-dot-org/pull/54943), we also added the last level of the Dance AI tutorial to the list of uncached unit level paths because it was project backed and used some dynamic HTML content needed for the export feature. This year, since the entire tutorial is Lab2, we should be able to cache every level, even though all Music Lab levels in the progression are project-backed (because we fetch user-specific project and level data after page load). Just confirming that this understanding is correct.

Reply:
Sounds good to me.